driver/src/main/scala/org/mongodb/scala/ReadPreference.scala

```diff
@@ -63,6 +66,38 @@ object ReadPreference {
   def nearest(): ReadPreference = JReadPreference.nearest()
 
   /**
+   * Gets a read preference that forces reads to the primary if available, otherwise to a secondary.
+   *
+   * @param maxStaleness the max allowable staleness of secondaries.  A zero duration indicates the absence of a maximum.
+   * @return ReadPreference which reads primary if available.
+   */
```

Comment:
Do all these need "@since" and " @note Requires MongoDB 3.4 or greater" annotations?